proposal/crystal0523/README.md

```diff
@@ -0,0 +1,42 @@
+# Fourier Transform for Image Processing
+## Basic Information
+* GitHub repository: https://github.com/crystal0523/nsdhw_21au/tree/crystal0523-proposal-submission/proposal/crystal0523
```

Comment:
You need a repository to house your code.  Fork of the homework repository cannot be used.